benchmark/crypto/oneshot-sign-verify.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+const common = require('../common.js');
+const crypto = require('crypto');
+const fs = require('fs');
+const path = require('path');
+const fixtures_keydir = path.resolve(__dirname, '../../test/fixtures/keys/');
+const pem = [{
+  publicKey: fs.readFileSync(`${fixtures_keydir}/ed25519_public.pem`)
+    .toString(),
+  privateKey: fs.readFileSync(`${fixtures_keydir}/ed25519_private.pem`)
+    .toString()
+}];
+
+const n = 1e3;
+
+for (let index = pem.length; index < n; index++) {
```

Comment:
Isn't this something that should be moved to `main()` since the *benchmark parameter* `n` is referenced when determining the index of the value in `pem`/`keyObjects` to pass to `crypto.sign()`/`crypto.verify()`?

Reply:
good point, i've cleaned up the benchmark code.